lib/zfs.py

```diff
@@ -1,51 +1,50 @@
 import os
 
 from shlex import split
+from subprocess import call
 from subprocess import check_call
 from subprocess import check_output
 
-from storagepool import StoragePool
+from storage_pool import StoragePool
 
 
 class ZfsPool(StoragePool):
-    '''The class to create a zfs storage pool.'''
+    '''The class to create raidz zfs storage pools from a list of devices.
+    The raidz group is a variation on RAID-5 that allows for better
+    distribution of parity and eliminates the "RAID-5 write hole" (in
+    which data and parity become inconsistent after a power loss). Data
+    and parity is striped across all disks within a raidz group.'''
 
     # The devices should be a list of strings.
     devices = []
     # The pool name must begin with a letter, and can only contain alphanumeric
     # characters as well as underscore ("_"), dash ("-"), period ("."), colon
     # (":"), and space (" "). The pool names "mirror", "raidz", "spare" and
     # "log" are reserved, as are names beginning with  the  pattern  "c[0-9]".
-    pool_name = 'zfs-storage-pool'
+    pool_name = 'juju-zfs-pool'
```

Comment:
I removed the pool_name because there is no use case for naming pools differently, so I decided not to provide the option. I could add it back in if you think it is necessary, or can give me a use case where we would need that.